### PR TITLE
Add mozilliansorg_pocket_fin_ops to Pocket section

### PR DESCRIPTION
**Pull request by bheesham**
There's a new group in town: https://people.mozilla.org/a/pocket_fin_ops/

Jira: [IAM-1470](https://mozilla-hub.atlassian.net/browse/IAM-1470)

### CODE REVIEW

File: rules/awsSaml.js [hunk 1]
```diff
@@ -60,6 +60,7 @@ async function awsSaml(user, context, callback) {
         "mozilliansorg_pocket_dataanalytics",
         "mozilliansorg_pocket_datalearning",
         "mozilliansorg_pocket_developer",
+        "mozilliansorg_pocket_fin_ops",
```
Comment (dividehex) [inline rules/awsSaml.js:63]: Rules are gone; try actions instead

**Review by dividehex — CHANGES_REQUESTED** [review]

File: rules/awsSaml.js (hunk 1 above)
Comment (bheesham) [reply rules/awsSaml.js:63]: ;_;

Whoops

**Comment by bheesham** [comment]
I went ahead and rebased https://github.com/mozilla-iam/auth0-deploy/pull/481 as well.

**Review by dividehex — APPROVED** [review]
🍒 Approved

**Comment by bheesham** [comment]
Applied!

**Pull request merged**